src/App.js

```diff
@@ -3,14 +3,28 @@ import Homepage from "./components/Homepage"
 import Header from "./components/Header"
 import Jumbotron from './components/Jumbotron'
 import Footer from "./components/Footer"
+import Order_page from "./components/Order_page";
 
 const App = () => {
+  const [activeRestaurantName, setActiveRestaurantName] = useState("")
+  const [activeRestaurantId, setActiveRestaurantId] = useState(0)
+
+  function buttonSetter(name, id) {
+    setActiveRestaurantName(name)
+    setActiveRestaurantId(id)
+  }
+
+  function idNameResetter() {
```

Comment:
Same here, don't love the function name - not descriptive enough, something like `resetActiveRestaurant`